### ISSUE TITLE
Error: Could not find or load main class com.infosys.utilities.EntryHome

### ISSUE THREAD

**Issue by ctproject4** [Infosys/openIDP#16]
Hi,
We are getting the following error while executing ReportFetchUtil post execution script in pipeline.
Kindly help us. Thanks!

**Comment by jaskirat8** [comment]
@ctproject4  Please refrain from opening new issue. Request you to reply over last issue only. 
Also it seems custom build is having issues in your system. 
Are there any network restrictions for you ?

**Comment by ctproject4** [comment]
@jaskirat8 Yes, I think. Network restrictions might be there due to corporate firewall. Can you help me communicating those restrictions with my companies IT team. If any ports or websites need to be unblocked. Also replying over last issue. Thank you!

**Comment by jaskirat8** [comment]
@ctproject4  for corporate scenarios we have provided exhaustive list off urls at [this ](https://github.com/Infosys/openIDP/wiki/Installation) Wiki Page.
Also build script supports proxy options if that available. You can look for require options in the list under section **Additional notes on setup script (build.sh) variables**

**Comment by krishnakanthbn** [comment]
@ctproject4 Hi,

Can you please let us know if you are still facing issues? If so, please share the logs/screenshots.

**Comment by ctproject4** [comment]
@krishnakanthbn As mentioned by @jaskirat8 that this may be caused due to corporate firewall issue, I also implemented this project on AWS outside the organisation. But still the same thing was happening. But I reconnected the slave via SSH connection and now its working fine. So you can close the ticket. Thanks btw!
